### DISCUSSION TITLE
switch#state=4 pace

### DISCUSSION THREAD

**Discussion by BusinessClaes** [arendst/Tasmota#14272] [Q&A] [answered]
As far as I understand, having a switch with Switchmode 11 fires the event switch#state=4 continuously as long as the switch is long-pressed.

But there is no way to control the pace with which the events state=4 are coming. Wouldn’t it be a good idea to be able to control it? It seems now that the events are coming as often as allowed by the processing power, which I don’t think is optimal (the pace is both unknown and uneven, depending on which other activity the processor is taking care of).

Wouldn’t it be possible to define some value with the desired pace in tenth of a second? And if the value is 0, then the events state=4 fire at max processing capacity.

**Comment by sfromis** [comment, marked as answer]
I see nothing to suggest that the speed depends on processing power, but instead a persistent pattern of ~1.6 seconds in between, as long as you have the default `SetOption32`. You can get a different speed changing that option.
